include/openssl/srp.h.in

```diff
@@ -92,81 +101,108 @@ typedef struct SRP_gN_st {
 -}
 
 
+OSSL_DEPRECATEDIN_3_0
 SRP_VBASE *SRP_VBASE_new(char *seed_key);
+OSSL_DEPRECATEDIN_3_0
 void SRP_VBASE_free(SRP_VBASE *vb);
+OSSL_DEPRECATEDIN_3_0
 int SRP_VBASE_init(SRP_VBASE *vb, char *verifier_file);
 
+OSSL_DEPRECATEDIN_3_0
 int SRP_VBASE_add0_user(SRP_VBASE *vb, SRP_user_pwd *user_pwd);
 /* This method ignores the configured seed and fails for an unknown user. */
-# ifndef OPENSSL_NO_DEPRECATED_1_1_0
```

Comment:
Should this be moved after the OSSL_NO_DEPRECATED_3_0 ifndef block?

Reply:
Done.